torchvision/transforms/v2/_container.py

```diff
@@ -100,14 +100,15 @@ def _extract_params_for_v1_transform(self) -> Dict[str, Any]:
         return {"transforms": self.transforms, "p": self.p}
 
     def forward(self, *inputs: Any) -> Any:
-        sample = inputs if len(inputs) > 1 else inputs[0]
+        needs_unpacking = len(inputs) > 1
```

Comment:
This got the same treatment as `transforms.Compose` in #7758. TL;DR: `RandomApply` now has the same UX as `Compose` in terms of passing packed or unpacked inputs.